### DISCUSSION TITLE
Recommended way to use Prisma in a workspaces/monorepo setup?

### DISCUSSION THREAD

**Discussion by ekafyi** [prisma/prisma#13943] [Q&A]
Hi all,

Looking for references/inspiration on using Prisma in a yarn workspaces (or similar) "monorepo" project setup, for example https://github.com/belgattitude/nextjs-monorepo-example

Prisma official blog has several posts on usage examples (eg. https://www.prisma.io/blog/fullstack-nextjs-graphql-prisma-3-clxbrcqppv), but I couldn't find any example in a workspaces/monorepo project.

Any other public starter templates or examples?

**Comment by nurul3101** [comment]
Hey @ekafyi 👋 
We have this simple example of using Prisma in the yarn workspace: [Prisma Monorepo Example](https://github.com/nikolasburk/monorepo-prisma)
Can you have a look at it? It's pretty simple but might be helpful to you.

**Reply by m1212e** [reply]
So this would require every project in the workspace to run on the same db schema, right?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
